docs/guides/sss_override.rst

```diff
@@ -0,0 +1,30 @@
+Local override (sss_override) users and groups
+##############################################
+
+Class :class:`sssd_test_framework.utils.sss_override.SSSOverrideUtils` provides an
+API to manage local overrides for users and groups. Also known as ID views, instead
+of being stored in IPA/LDAP server the override data is stored locally in SSSD's
+cache.
+
+SSSD must be running for this API to work.
+
+.. code-block:: python
+    :caption: Examples
+
+    @pytest.mark.topology(KnownTopology.LDAP)
+    @pytest.mark.topology(KnownTopology.AD)
+    def test_local_overrides__user(client: Client, provider: GenericProvider):
+        provider.user("user1").add(
+        uid=999011,
+        gid=999011,
+        home="/home/user1",
+        gecos="user",
+        shell="/bin/bash",
+        password="Secret123"
+    )
+    client.sssd.start()
+    client.sss_override.user("user1").add(name="o-user1")
+    client.sssd.restart()
+
+    assert client.ssh("user1", "Secret123")
+    assert client.ssh("o-user1", "Secret123")
```

Comment:
Correct indentation. Always execute the test that you add as an example, because if the documentation will not contain working examples, we are doomed.